src/components/Medicine/MedicationAdministration/AdministrationTab.tsx

```diff
@@ -218,11 +220,19 @@ const MedicationRow: React.FC<MedicationRowProps> = ({
   );
 
   return (
-    <React.Fragment>
+    <React.Fragment key={medication.id}>
```

Comment:
does this solve anything? 🤔 

```suggestion
    <React.Fragment>
```